packages/react-avatar/src/components/Avatar/useAvatarStyles.ts

```diff
@@ -0,0 +1,472 @@
+import { ax, makeStyles } from '@fluentui/react-make-styles';
+import { AvatarState } from './Avatar.types';
+
+//
+// TODO: All animation constants should go to theme or globals?
+//
```

Comment:
Is there an issue to follow up on this TODO?

Reply:
```suggestion
//
// TODO: All animation constants should go to theme or globals?
// https://github.com/microsoft/fluentui/issues/16372#issuecomment-778240665
```

Good call 👍 Covered in follow up items.